src/ert/analysis/configuration.py

```diff
@@ -68,8 +74,14 @@ def observation_config(self) -> List[Tuple[str, Optional[List[int]]]]:
         ]
 
 
+if TYPE_CHECKING:
+    ConstrainedUpdateStepList = List[UpdateStep]
+else:
+    ConstrainedUpdateStepList = conlist(UpdateStep, min_items=1)
+
+
```

Comment:
Do we need two if these blocks in the same file ?

Reply:
I don't see how I can do it with just one. Do you know of a way?

Reply:
Is it not similar to ie:

https://github.com/equinor/ert/blob/589da5f5751d9a51d77deb664c734f5edcf92dec/src/ert/analysis/_es_update.py#L35

Where you define typing on the top of the file or are you trying to do something else?

Reply:
It is not the same as I am here defining both the type for static type-checking and a `conlist` for pydantic.
If it was just for type-checking, then I could get away with just one if.